app/views/modules/_header.html.erb

```diff
@@ -53,10 +53,14 @@ Unless required by applicable law or agreed to in writing, software distributed
           </div><!-- /.btn-group -->
         <% else %>
           <div class="btn-group login_button pull-right">
-            <%= link_to(new_user_session_path, class: "btn btn-default btn-group", title: "Login") do %>
-              <span class="glyphicon glyphicon-log-in" aria-hidden="true"></span>
-              <%= t('ualberta.era_av_sign_in') %>
-            <% end %>
+            <%- Devise.omniauth_providers.each do |provider| %>
```

Comment:
I'm having trouble reproducing the Avalon 5 approach to adding the Shibboleth authentication path to the log-in link (See [Avalon 5]https://github.com/ualbertalib/avalon/blob/master/app/views/devise/sessions/new.html.erb#L24-L29) ) & e86bf82. 

Is the Avalon 5 approach better (I can't find variables `devise_mapping` or `resource_class`) or the following approach implemented in the referenced block?

Reply:
Remove 'do' and 'if' blocks and use `omniauth_authorize_path(:user, :shibboleth)` instead?

Reply:
`user_{provider}_omniauth_authorize_path` from the [README](https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview) for example
```
<%= link_to "Sign in with Shibboleth", user_shibboleth_omniauth_authorize_path %>
```